src/dotnet/APIView/APIView/Model/CodeFile.cs

```diff
@@ -52,10 +45,18 @@ public string VersionString
         public string PackageVersion { get; set; }
         public string CrossLanguagePackageId { get; set; }
         public CodeFileToken[] Tokens { get; set; } = Array.Empty<CodeFileToken>();
+        // APIForest will be removed once server changes are added to dereference this property
         public List<APITreeNode> APIForest { get; set; } = new List<APITreeNode>();
         public List<CodeFileToken[]> LeafSections { get; set; }
         public NavigationItem[] Navigation { get; set; }
         public CodeDiagnostic[] Diagnostics { get; set; }
+        public string ParserVersion
```

Comment:
Why ParserVersion? Will `VersionString` eventually be removed?

Reply:
> Why ParserVersion? Will `VersionString` eventually be removed?

Yes. VersionString has always been confusing and misleading. It was interpreted as package version in the past. Renamed the property to make it convey the usage easily.